Formula/opencoarrays.rb

```diff
@@ -13,7 +14,11 @@ class Opencoarrays < Formula
     sha256 "fa3e7e32db42b67198043aa0c22dcd6f814b234897d2cfd8c9195fe4fd1e017b" => :yosemite
   end
 
-  option "without-test", "Skip build time tests (not recommended)"
+  # As long as there are known failures with Homebrew's default compiler,
+  # we turn the tests off by default
+  # https://github.com/sourceryinstitute/OpenCoarrays/issues/374
+  # option "without-test", "Skip build time tests (not recommended)"
+  option "with-test", "Perform build time tests"
```

Comment:
We should have a release out later today that "fixes" the build time tests with GCC 7. By fixes I mean that we've added a flag for OpenCoarrays developers so that by default tests for known regressions (`convert-before-put`) and partially implemented features (`async_comp_alloc`) are turned __*OFF*__ and are only enabled when `OPENCOARRAYS_DEVELOPER=ON` in the environment or the advanced CMake option `RUN_DEVELOPER_TESTS` is `ON`.

You can see the proposed changes at https://github.com/sourceryinstitute/OpenCoarrays/pull/375